### PR TITLE
(CAT-1434) Update README

### PR DESCRIPTION
**Pull request by GSPatton**
Removed outdated compatability warning since Puppet 6 is not supported by any tool. Added development section taken from https://github.com/puppetlabs/puppetlabs-concat#development detailing how users can run acceptance tests, raise PRs/Issues and contact support

## Summary
Provide a detailed description of all the changes present in this pull request.

## Additional Context
Add any additional context about the problem here. 
- [ ] Root cause and the steps to reproduce. (If applicable)
- [ ] Thought process behind the implementation.

## Related Issues (if any)
Mention any related issues or pull requests.

## Checklist
- [ ] 🟢 Spec tests.
- [ ] 🟢 Acceptance tests.
- [ ] Manually verified.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,12 +7,6 @@
 Puppet Lint tests Puppet code against the recommended [Puppet language style
 guide](http://puppet.com/docs/puppet/latest/style_guide.html). Puppet Lint validates only code style; it does not validate syntax. To test syntax, use Puppet's `puppet parser validate` command.
 
-## Compatibility warning
```
Comment (ekohl) [inline README.md:10]: This is useful information. Please keep it in. Perhaps update it to make it clear it is current.

**Review by ekohl — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -297,6 +291,18 @@ If you find a bug in Puppet Lint or its results, please create an issue in the
 [repo issues tracker](https://github.com/puppetlabs/puppet-lint/issues/). Bonus
 points will be awarded if you also include a patch that fixes the issue.
 
+## Development
+
+Acceptance tests for this tool leverage [puppet_litmus](https://github.com/puppetlabs/puppet_litmus).
```
Comment (ekohl) [inline README.md:296]: I really wish it didn't. Litmus an aweful tool for this and I wish https://github.com/puppetlabs/puppet-lint/pull/89 got a proper response for maintainers.

Comment (GSPatton) [reply README.md:296]: Hi @ekohl. I'll make a note of this and bring it to the team for discussion

File: README.md [hunk 3]
```diff
@@ -297,6 +295,18 @@ If you find a bug in Puppet Lint or its results, please create an issue in the
...
+To run the acceptance tests follow the instructions [here](https://github.com/puppetlabs/puppet_litmus/wiki/Tutorial:-use-Litmus-to-execute-acceptance-tests-with-a-sample-module-(MoTD)#install-the-necessary-gems-for-the-module).
+You can also find a tutorial and walkthrough of using Litmus and the PDK on [YouTube](https://www.youtube.com/watch?v=FYfR7ZEGHoE).
+
+If you run into an issue with this tool or would like to request a feature you can [raise a PR](https://github.com/puppetlabs/puppet-lint/pulls) with your suggested changes. Alternatively, you can [raise a Github issue](https://github.com/puppetlabs/puppet-lint/issues) with a feature request or to report any bugs.
+Every other Tuesday the DevX holds [office hours](https://puppet.com/community/office-hours) in the [Puppet Community Slack](http://slack.puppet.com/), where you can ask questions about this and any other supported tools.
```
Comment (jordanbreen28) [inline README.md:305]:
```suggestion
Every other Tuesday the DevX team holds [office hours](https://puppet.com/community/office-hours) in the [Puppet Community Slack](http://slack.puppet.com/), where you can ask questions about this and any other supported tools.
```

File: README.md [hunk 4]
```diff
@@ -297,6 +295,18 @@ If you find a bug in Puppet Lint or its results, please create an issue in the
...
+This session runs at 15:00 (BST) for about an hour.
+
+If you have problems getting this module up and running, please [contact Support](http://puppetlabs.com/services/customer-support).
```
Comment (jordanbreen28) [inline README.md:308]:
```suggestion
If you have problems getting this tool up and running, please [contact Support](http://puppetlabs.com/services/customer-support).
```

**Review by jordanbreen28 — APPROVED** [review]

**Pull request merged**